maintainers/maintainer-list.nix

```diff
@@ -4696,6 +4696,12 @@
     github = "d4ilyrun";
     githubId = 34611103;
   };
+  d4rk = {
+    name = "Anoop Menon";
+    email = "d4rk@blackvoltage.org";
+    github = "d4rk";
+    githubId = 22163;
+  };
```

Comment:
This should ideally be in a separate commit. And the other commit should say "init at 0-unstable-2024-10-19"

Reply:
Ah sorry, split the two and reworded the other one.